### PR TITLE
:running: Update Go 1.13

### PR DESCRIPTION
**Pull request by vincepri**
Signed-off-by: Vince Prignano <vincepri@vmware.com>

### CODE REVIEW

**Comment by vincepri** [comment]
Waiting on https://github.com/kubernetes/test-infra/pull/15620

File: Dockerfile [hunk 1]
```diff
@@ -13,7 +13,7 @@
 # limitations under the License.
 
 # Build the manager binary
-FROM golang:1.12.10 as builder
+FROM golang:1.13.4 as builder
```
Comment (detiber) [inline Dockerfile]: Should this be 1.13.5 instead?

File: Makefile [hunk 2]
```diff
@@ -294,7 +294,7 @@ release-binary: $(RELEASE_DIR)
 		-e GOARCH=$(GOARCH) \
 		-v "$$(pwd):/workspace$(DOCKER_VOL_OPTS)" \
 		-w /workspace \
-		golang:1.12.10 \
+		golang:1.13.4 \
```
Comment (detiber) [inline Makefile]: Should this be 1.13.5 instead?

File: cmd/example-provider/Dockerfile [hunk 3]
```diff
@@ -13,7 +13,7 @@
 # limitations under the License.
 
 # Build the manager binary
-FROM golang:1.12.10 as builder
+FROM golang:1.13.4 as builder
```
Comment (detiber) [inline cmd/example-provider/Dockerfile]: Should this be 1.13.5 instead?

File: test/infrastructure/docker/Dockerfile [hunk 4]
```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM golang:1.12.10 as builder
+FROM golang:1.13.4 as builder
```
Comment (detiber) [inline test/infrastructure/docker/Dockerfile]: Should this be 1.13.5 instead?

File: test/infrastructure/docker/Dockerfile.dev [hunk 5]
```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM golang:1.12.10
+FROM golang:1.13.4
```
Comment (detiber) [inline test/infrastructure/docker/Dockerfile.dev]: Should this be 1.13.5 instead?

File: hack/ensure-go.sh [hunk 6]
```diff
@@ -31,7 +31,7 @@ EOF
   local go_version
   IFS=" " read -ra go_version <<< "$(go version)"
   local minimum_go_version
-  minimum_go_version=go1.12.1
+  minimum_go_version=go1.13.4
```
Comment (detiber) [inline hack/ensure-go.sh]: Do we specifically need to ensure minimum is 1.13.4 here? or can we just ensure it is at least 1.13.0 for the local compiler?

Comment (vincepri) [reply hack/ensure-go.sh]: Makes sense yeah

**Comment by vincepri** [comment]
Fixed, ptal

**Pull request merged**